pkgs/applications/editors/manuskript/default.nix

```diff
@@ -28,11 +28,22 @@ python3Packages.buildPythonApplication rec {
 
   buildPhase = "";
 
-  installPhase = ''
-    mkdir -p $out/share/${pname}
-    cp -av  bin/ i18n/ libs/ manuskript/ resources/ icons/ $out
-    cp -r sample-projects/ $out/share/${pname}
-  '';
+  installPhase = (
+    let
+      desktopItem = makeDesktopItem {
+        name = "Manuskript";
+        exec = "manuskript";
+        desktopName = "Manuskript";
+        genericName = "Manuskript";
+        categories = "Office;";
+      };
+    in
+      ''
+        mkdir -p $out/share/${pname}
+        cp -av  bin/ i18n/ libs/ manuskript/ resources/ icons/ $out
+        cp -a ${desktopItem}/share/applications $out/share
```

Comment:
Please use copyDesktopItems instead of copying manually.

Reply:
I tried that, but it doesn't seem to work by just putting it on the nativeBuildInputs when a manual `installPhase` is set, nor could I find any documentation or examples of other derivations doing that to copy. Is there some documentation I'm missing?

Reply:
You need to add runHooks to the custom installPhase https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/bucklespring/default.nix#L40-L48.